etc/collectd.d/statsd.conf

```diff
@@ -0,0 +1,30 @@
+LoadPlugin statsd
```

Comment:
This file needs to be a template with a copyright header (see, e.g., [templates/memcached.conf.jinja](https://github.com/Stackdriver/stackdriver-agent-service-configs/blob/master/templates/memcached.conf.jinja)). Please make this 2 commits: one to add the template, and one to generate the actual config.